common/test/acceptance/pages/lms/video/video.py

```diff
@@ -346,6 +391,22 @@ def captions_text(self):
         return ' '.join(subs)
 
     @property
+    def closed_captions_text(self):
+        """
+        Extract closed captioning text.
+
+        Returns:
+            str: closed captions Text.
+
+        """
+        self.wait_for_closed_captions()
+
+        closed_captions_selector = self.get_element_selector(CSS_CLASS_NAMES['closed_captions'])
+        subs = self.q(css=closed_captions_selector).html
```

Comment:
This test was duplicated from the one above for the transcripts. I believe `subs` here refers to the text, either in the transcript or in the caption (whichever the test is checking).